### PR TITLE
Sandeep prettier

### PR DESCRIPTION
**Pull request by sandeep-deriv**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `Fixes #1, Fixes #2` 
| Patch: Bug Fix?          |
| Major: Breaking Change?  |
| Minor: New Feature?      |
| Tests Added + Pass?      | Yes
| Does not reduce coverage?| Yes
| Any Dependency Changes?  |

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+www/*
+xml-tests/*
+templates/*
+static/*
+examples/*
+docs/*
+cli-examples/*
```
Comment (yashim-deriv) [inline .prettierignore:7]:
```suggestion
cli-examples/*
node_modules/*
lib/*
```

**Review by yashim-deriv — COMMENTED** [review]
I think you missed some of the directories

File: .prettierignore (hunk 1 above)
Comment (sandeep-deriv) [reply .prettierignore:7]: sure, let me add these

**Review by yashim-deriv — APPROVED** [review]
LGTM~

**Pull request merged**